theme/scripts/cart.ts

```diff
@@ -284,27 +280,30 @@ export default(function() {
         }
       })
     },
+  
+    handleProductAddedToCart() {
+      const desktopProductAddToCartButton = document.querySelector(Cart.SELECTORS.desktopProductAddToCartButton);
+      const mobileProductAddToCartButton = document.querySelector(Cart.SELECTORS.mobileProductAddToCartButton);
 
-    //TO-DO (#178): This function should check when the add to cart button is clicked and use a setTimeout to change the text to "Added to cart" and then revert back to "Add to cart" after a few seconds.
-    handleAddToCartButton(variantId: string, action: Action) {
-      const addToCartButton = document.querySelector(Cart.SELECTORS.addToCartButton);
-      
-      if (!addToCartButton) return;
-      
-      const addToCartVariantId = addToCartButton.getAttribute(Cart.ATTRIBUTES.variantId);
-
-      if (addToCartVariantId === variantId) {
-        if (action === Action.ADD) {
-          addToCartButton.textContent = Cart.CONSTANTS.addedToCart;
-          addToCartButton.setAttribute(Cart.ATTRIBUTES.disabled, Cart.ATTRIBUTES.disabled);
-        }
+      if (desktopProductAddToCartButton) {
+        Cart.handleProductAddedToCartButtonStyling(desktopProductAddToCartButton);
+      }
 
-        if (action === Action.REMOVE) {
-          addToCartButton.textContent = Cart.CONSTANTS.addToCart;
-          addToCartButton.removeAttribute(Cart.ATTRIBUTES.disabled);
-        }
+      if (mobileProductAddToCartButton) {
+        Cart.handleProductAddedToCartButtonStyling(mobileProductAddToCartButton);
       }
     },
+    
+    handleProductAddedToCartButtonStyling(button: Element) {
+      button.textContent = Cart.CONSTANTS.addedToCart;
+      button.setAttribute(Cart.ATTRIBUTES.disabled, Cart.ATTRIBUTES.disabled);
+
+      setTimeout(() => {
+        button.textContent = Cart.CONSTANTS.addToCart;
+        button.removeAttribute(Cart.ATTRIBUTES.disabled);
+      }, 2000);  
+    },
+
```

Comment:
I made a separate function to handle the styling -- not sure if necessary and/or more confusing!

Reply:
Separate function is easier to read :)